modules/oadp-configuring-aws-bsl.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-bsl.adoc_{context}"]
+= Configuring the backup storage location using AWS
+
+You can configure the {aws-short} backup storage location (BSL) as shown in the following example.
+
+.Prerequisites
+
+* You have created an object storage bucket using {aws-short}.
+* You have installed the {oadp-short} Operator.
+
+.Procedure
+
+. Configure the BSL custom resource (CR) as shown in the following example:
++
+.Example backup storage location
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: BackupStorageLocation
+metadata:
+  name: default
+  namespace: openshift-adp
+spec:
+  provider: aws # <1>
+  objectStorage:
+    bucket: <bucket_name> # <2>
+    prefix: <bucket_prefix> # <3>
+  credential: # <4>
+    key: cloud # <5>
+    name: cloud-credentials # <6>
+  config:
+    region: <bucket_region> # <7>
+    s3ForcePathStyle: "true" # <8>
+    s3Url: <s3_url> # <9>
+    publicUrl: <public_s3_url> # <10>
+    serverSideEncryption: AES256 # <11>
+    kmsKeyId: "50..c-4da1-419f-a16e-ei...49f" # <12>    
+    customerKeyEncryptionFile: "/credentials/customer-key" # <13>
+    signatureVersion: "1" # <14>
+    profile: "default" # <15>
+    insecureSkipTLSVerify: "true" # <16>
+    enableSharedConfig: "true" # <17>
+    tagging: "" # <18>
+    checksumAlgorithm: "CRC32" # <19>
+----
+<1> The name of the object store plugin. In this example, the plugin is `aws`. This field is required.
+<2> The name of the bucket in which to store backups. This field is required.
+<3> The prefix within the bucket in which to store backups. This field is optional.
+<4> The credentials for the backup storage location. You can set custom credentials. If custom credentials are not set, the default credentials' secret is used.
+<5> The `key` within the secret credentials' data.
+<6> The name of the secret containing the credentials.
+<7> The AWS region where the bucket is located. Optional if s3ForcePathStyle is false.
+<8> A boolean flag to decide whether to use path-style addressing instead of virtual hosted bucket addressing. Set to "true" if using a storage service such as MinIO or NooBaa. This is an optional field. The default value is `false`.
```

Comment:
So this is a little tricky, but even if the value is "true" with quotes in the YAML, it still needs to be in backticks because it's part of a file:
```suggestion
<8> A boolean flag to decide whether to use path-style addressing instead of virtual hosted bucket addressing. Set to `"true"` if using a storage service such as MinIO or NooBaa. This is an optional field. The default value is `false`.
```

Reply:
Or if this was simply meant to be backticks without quotes, you can do that as well.